### PR TITLE
Microsoft.StorageSync 2019-06-01 API version

### PR DESCRIPTION
**Pull request by anpint**
### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [x] I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [x] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [x] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

**Comment by anpint** [comment]
@Juliehzl, for your information, this PR contains the 2019-06-01 API version swagger generated through the OpenAPIHub, and updated with the changes that have been already approved in: https://github.com/Azure/azure-rest-api-specs/pull/7161.

File: specification/storagesync/resource-manager/readme.md [hunk 1]
```diff
@@ -26,18 +26,27 @@ These are the global settings for the Storage Sync API.
 
 ``` yaml
 openapi-type: arm
-tag: package-2019-03-01
+tag: package-2019-06
```
Comment (ryansbenson) [inline specification/storagesync/resource-manager/readme.md]: shouldn't this be 2019-06-01?

**Comment by ryansbenson** [comment]
Approved from ARMs side.

SDK - Please review the readme change.

File: specification/storagesync/resource-manager/readme.md (hunk 1 above)
Comment (anpint) [reply specification/storagesync/resource-manager/readme.md]: Thanks for pointing it out @ryansbenson 
I'll fix it.

Comment (anpint) [reply specification/storagesync/resource-manager/readme.md]: @Juliehzl please approve the following PR:
https://github.com/Azure/azure-rest-api-specs/pull/7413

Looks like I don't have permission to push to this branch, so I had to create a PR to fix the issue in the readme file.

**Comment by yungezz** [comment]
hi @Juliehzl is the PR ok to merge?

**Comment by anpint** [comment]
@yungezz It looks like @Juliehzl has been unresponsive for many days now. Are you able to help completing the PR?

File: specification/storagesync/resource-manager/readme.md [hunk 2]
```diff
@@ -26,18 +26,27 @@ These are the global settings for the Storage Sync API.
...
+```yaml $(tag) == 'package-2019-06'
+input-file:
+  - Microsoft.StorageSync/stable/2019-06-01/storagesync.json
+```
 ### Tag: package-2019-03-01
 
 These settings apply only when `--tag=package-2019-03-01` is specified on the command line.
 
-```yaml $(tag) == 'package-2019-03-01'
+``` yaml $(tag) == 'package-2019-03-01'
```
Comment (sergey-shandar) [inline specification/storagesync/resource-manager/readme.md]: No need to add a space.

File: specification/storagesync/resource-manager/readme.md [hunk 3]
```diff
@@ -155,7 +164,7 @@ See configuration in [readme.go.md](./readme.go.md)
 
 See configuration in [readme.java.md](./readme.java.md)
 
-## Multi-API/Profile support for AutoRest v3 generators 
+## Multi-API/Profile support for AutoRest v3 generators
 
 AutoRest V3 generators require the use of `--tag=all-api-versions` to select api files.
```
Comment (sergey-shandar) [inline specification/storagesync/resource-manager/readme.md]: IMHO, the multiapi profile should also have the new version.

File: specification/storagesync/resource-manager/readme.md (hunk 1 above)
Comment (sergey-shandar) [reply specification/storagesync/resource-manager/readme.md]: This is not fixed yet.

**Review by sergey-shandar — CHANGES_REQUESTED** [review]

File: specification/storagesync/resource-manager/readme.md (hunk 2 above)
Comment (anpint) [reply specification/storagesync/resource-manager/readme.md]: I didn't modify this myself (likely code generation made the changes). I can fix it anyways.

File: specification/storagesync/resource-manager/readme.md (hunk 3 above)
Comment (anpint) [reply specification/storagesync/resource-manager/readme.md]: Good point. However, I see that the there is some text there that reads: "This block is updated by an automatic script. Edits may be lost!", so I feel maybe it should have been automatically updated. I'll add the new version there anyways.

File: specification/storagesync/resource-manager/readme.md (hunk 1 above)
Comment (anpint) [reply specification/storagesync/resource-manager/readme.md]: I don't have permissions to push to this branch, so the fixes are going to this PR:
https://github.com/Azure/azure-rest-api-specs/pull/7413

**Review by sergey-shandar — APPROVED** [review]

**Pull request merged**